api/src/main/java/org/apache/iceberg/variants/Variant.java

```diff
@@ -22,6 +22,10 @@
 
 /** A variant metadata and value pair. */
 public interface Variant {
```

Comment:
In TypeWithShemaVisitor.java, there is another place that we need to change. Can you also update there?

```
      } else if (annotation instanceof LogicalTypeAnnotation.VariantLogicalTypeAnnotation || (iType != null && iType.isVariantType())) {
        // when Parquet has a VARIANT logical type, use it here
        return visitVariant(iType.asVariantType(), group, visitor);
      }
```

Reply:
Updated, thanks for checking @aihuaxu